calculator.js

```diff
@@ -1,5 +1,88 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+// This line "loads" the prompt package and puts it into a variable we can use called prompt
+// Lines like this (that use require()) typically go on the top of the file
+const prompt = require('prompt');
```

Comment:
Nice!

Just note that in the future add `node_modules` to your `.gitignore` file.